resources/testharness.js

```diff
@@ -693,6 +701,9 @@ policies and contribution forms [3].
         }
         var end_wait = function() { tests.end_wait(); };
         if (tests.file_is_test) {
+            // Register a "done" callback to ensure the invocation of
+            // `end_wait` is deffered until after test "cleanup" logic has
```

Comment:
s/deffered/deferred/

Reply:
I removed this comment as part of the refactoring mentioned above.